src/import_edf.jl

```diff
@@ -365,10 +396,10 @@ following transformations:
 
 See the OndaEDF README for additional details regarding EDF formatting expectations.
 """
-function edf_to_onda_samples(edf::EDF.File; custom_extractors=())
+function edf_to_onda_samples(edf::EDF.File; custom_extractors=STANDARD_EXTRACTORS)
     EDF.read!(edf)
     edf_samples = Samples[]
-    for extractor in Iterators.flatten((STANDARD_EXTRACTORS, custom_extractors))
+    for extractor in custom_extractors
```

Comment:
can you explain the motivation for making custom_extractors replace instead of extend the standards?  I thought in a previous version of this PR you'd prepended them instead of postpending, which seems like a better choice to me but something clearly made you change your mind.